CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+project(wxFormBuilder)
+cmake_minimum_required(VERSION 3.0)
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMakeModules)
+set(CMAKE_CXX_STANDARD 17)
+
+find_package(wxWidgets 3.0 REQUIRED gl aui adv html core net base xml stc richtext propgrid ribbon xrc media)
+include(${wxWidgets_USE_FILE})
+
+set(CPACK_PACKAGE_VERSION_MAJOR 3)
+set(CPACK_PACKAGE_VERSION_MINOR 6)
+set(CPACK_PACKAGE_VERSION_PATCH 0)
+set(CPACK_RESOURCE_FILE_LICENSE ${PROJECT_SOURCE_DIR}/output/license.txt)
+set(CPACK_WIX_UPGRADE_GUID B5AAB525-34D5-4B12-B6F2-136C8ED890C9)
+
+if(APPLE)
+    set(CPACK_SYSTEM_NAME macOS)
+endif()
+
+include(CPack)
+
+if(MSVC)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
```

Comment:
Has been superseded by `add_compile_definitions`, see https://cmake.org/cmake/help/v3.16/command/add_definitions.html

Reply:
which cmake version is targeted? i also don't use MSVC, so its stupid when i change it, as i can't test it